swarm/src/main/resources/swarm/swarm.bom

```diff
@@ -446,6 +446,10 @@ brooklyn.catalog:
           default: true
 
       brooklyn.config:
+        name: swarm.create.shared.securitygroup
```

Comment:
I don't think this will work. It creates four config keys, `name`, `label`, `type`, and `default`.